android/src/com/mozilla/vpn/VPNService.kt

```diff
@@ -176,13 +220,14 @@ class VPNService : android.net.VpnService() {
         val intent = Intent(this, activity)
         val pendingIntent = PendingIntent.getActivity(this, 0, intent, 0)
 
-        val builder = NotificationCompat.Builder(this, NOTIFICATION_CHANNEL_ID)
+        mNotificationBuilder
```

Comment:
I would like to have the notification controlled by the app, but I guess, at this point, the app has not been executed yet, right?

Reply:
Yep, i.e. connect-on-boot or always-on are enabled the service will get started from the OS without starting our qt code.